src/Microsoft.TemplateEngine.Edge/Settings/TemplateMatchInfo.cs

```diff
@@ -58,4 +62,39 @@ public void AddDisposition(MatchInfo newDisposition)
 
         public bool IsPartialMatch => MatchDisposition.Any(x => x.Kind != MatchKind.Mismatch);
     }
+
+    internal class TemplateMatchInfo2 : Abstractions.TemplateFiltering.ITemplateMatchInfo
```

Comment:
Would this better be in a new file?

Reply:
the plan is to rename it to 'TemplateMatchInfo" when obsolete is removed. I'm not sure about new file therefore